README.md

```diff
@@ -11,28 +11,42 @@ A Model Context Protocol server for interacting with Appwrite's API. This server
 - [Installation](#installation)
 - IDE Integration:
   - [Claude Desktop](#usage-with-claude-desktop)
-  - [Zed](#usage-with-zed)
   - [Cursor](#usage-with-cursor)
+  - [Windsurf Editor](#usage-with-windsurf-editor)
 - [Local Development](#local-development)
 - [Debugging](#debugging)
 
-Currently, the server supports the following tools:
+## Configuration
 
-- [x] Databases
-- [x] Users
+> Before launching the MCP server, you must setup an [Appwrite project](https://cloud.appwrite.io/) and create an API key with the necessary scopes enabled.
 
-> Please note that adding a lot of tools exceeds the context window of the LLM. As a result, we will make available a curated list of tools that are most commonly used.
+Open the terminal in your working directory and run the following command:
 
-## Configuration
+### Linux and MacOS
```

Comment:
@adityaoberai I think its better to just store these values in a `.env` file and run `source .env` . This works on linux and mac.. see if you can find a similar alternative for windows

Reply:
I'm looking into alternatives for `source`